include/openssl/ssl.h.in

```diff
@@ -1501,6 +1501,8 @@ DECLARE_PEM_rw(SSL_SESSION, SSL_SESSION)
 # define SSL_get_max_proto_version(s) \
         SSL_ctrl(s, SSL_CTRL_GET_MAX_PROTO_VERSION, 0, NULL)
 
+const char* SSL_groupid_to_name(SSL *s, int group_id);
```

Comment:
Please add space after `char`

Reply:
And remove the space after `*`, i..e.

````
const char *SSL_groupid_to_name(SSL *s, int group_id);
````

Reply:
Done.